coc/ext/discordlinks/__init__.py

```diff
@@ -132,7 +132,7 @@ async def _request(self, method, url, *, token_request: bool = False, **kwargs):
                 return await self._request(method, url, **kwargs)
 
     async def _get_key(self):
-        if not self.key or self.key.expires_at < datetime.utcnow():
+        if not self.key or (self.key.expires_at and self.key.expires_at < datetime.utcnow()):
```

Comment:
Please wrap the `and` with parenthesis. The order of precedence is clear but we should not have our readers be literate in the order of precedence.